Notices.md

```diff
@@ -0,0 +1,58 @@
+# Notices
+
+## Code of Conduct
+
+Contact for Code of Conduct issues or inquires:
+
+* Christopher Atkins @ MISO
+* Tory McKeag @ GE Vernova
+
+## License Acceptance
+
+Per Community Specification License 1.0 Section 2.1.3.3, Licensees may indicate their acceptance of the Community Specification License by issuing a pull request to the Specification’s repository’s Notices.md file, including the Licensee’s name, authorized individuals' names, and repository system identifier (e.g. GitHub ID), and specification version.
+
+A Licensee may consent to accepting the current Community Specification License version or any future version of the Community Specification License by indicating "or later" after their specification version.
```

Comment:
I'm not sure if EasyCLA obviates the need for contributors to do this or not. @jmertic can you educate me on this?

Reply:
Hmm - that probably is a better @jorydotcom question - Jory can you advise here?